FlowCal/transform.py

```diff
@@ -118,6 +128,15 @@ def to_rfi(data, channels=None, max_range=None, amplification_type=None):
         the first element is zero, the amplification type is linear. If
         None, take `amplification_type` from
         ``data.amplification_type(channel)``.
+    amplifier_gain : float or list of floats, optional
+        The linear amplifier gain of the specified channel(s). Only used if
+        ``amplification_type[0]==0`` (linear amplifier). If None,
+        take `amplifier_gain` from ``data.amplifier_gain(channel)``. If
+        `data` does not contain ``amplifier_gain()``, use 1.0.
+    max_range : int, float, or list of int or float, optional
+        Maximum range, for each specified channel. Only needed if
+        ``amplification_type[0]!=0`` (log amplifier). If None, take
+        `max_range` from ``len(data.domain(channel))``.
```

Comment:
do you call it `max_range` because `range` is already a native Python keyword? `max_range` wasn't intuitive for me, was trying to figure out why you had to specify a max (like why was it bounded? and how did that factor into the `to_rfi()` transformation). I guess i'd prefer `amp_range` (consistent with the shortening of the other 2 amplifier parameters)